go-controller/pkg/network-attach-def-controller/network_manager.go

```diff
@@ -32,6 +32,8 @@ type networkManager interface {
 
 	// Stop the controller
 	Stop()
+
+	getNetwork(string) util.NetInfo
```

Comment:
nit: I would just do exported `GetNetwork` like the others

Reply:
i did this intentionally. exposing it outside of the package would allow another package to get the direct pointer to the value in the cache. As you said yesterday nadController and NM are pals, but we cant trust consumers outside of the package.

Reply:
The interface itself is not exported so it can't be used outside of the package. In these cases I usually export methods to tell me its being used within the package widely. But fair enough, I see how that's a personal perspective while yours is different and valid.